main.py

```diff
@@ -1,15 +1,12 @@
-from pypanther import PantherLogType, PantherSeverity, get_panther_rules, register
-from pypanther.wrap import exclude, include
-from pypanther.get import table_print
-
+from pypanther import LogType, Severity, get_panther_rules, get_rules, register
+from pypanther.get import print_rule_table
 from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_account_discovery import AWSCloudTrailAccountDiscovery
 from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_stopped import AWSCloudTrailStopped
 from pypanther.rules.aws_cloudtrail_rules.aws_console_root_login import AWSConsoleRootLogin
+from pypanther.wrap import exclude, include
 
-from rules.aws_rules.alb_high_400s import AWSALBHighVol400s
-from rules.custom_logs.ids_rules import HostIDSBaseRule, HostIDSMalware
-from helpers.cloud import prod_account_ids, account_lookup_by_id, update_account_id_tests
-
+import rules
```

Comment:
nit: isn't this import order wrong?

Reply:
It works as is. It does look funny though. Should it be before the others?

Reply:
The real answer is that it should be handled by the linter so that I don't have to think about it. But maybe the linter does handle it because looking at https://peps.python.org/pep-0008/#imports I think it's right. I guess it's rare to have local imports with no from

Reply:
Yeah. I think I have also noticed the `ruff` claims to be a drop in replacement for `isort` but I don't think it actually does as good of a job as `isort` does